### PR TITLE
Newsletter-219: Translate to French

### PR DESCRIPTION
**Pull request by Copinmalin**
[Update 2022-09-28-newsletter.md]
(https://github.com/Copinmalin/bitcoinops.github.io/commit/54ef28368aa6e96509f1b79f3de1c2ec13572efe)

Ready for review

### CODE REVIEW

File: _posts/fr/newsletters/2022-09-28-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    d'essayer une autre route qui peut qui peut avoir plus de sauts mais un coût
+    effectif plus faible, ou bien essayer la même canal à un coût plus élevé."
+
+    La proposition prévoit également des redevances négatives.  Par exemple,
+    une chaîne pourrait subventionner les paiements qui lui sont transmis
+    lorsqu'il a plus de 75 % de capacité sortante en ajoutant 1 satoshi pour
+    chaque tranche de 1 000 satoshis de valeur de paiement.  Les frais négatifs
+    pourraient être utilisés par les marchands pour inciter les autres à restaurer
+    la capacité entrante des canaux fréquemment utilisés pour recevoir des paiements.
+
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2022-09-28-newsletter.md]: Trailing space here. I pushed a commit to address. (you can see such errors locally running `make production`)

File: _posts/fr/newsletters/2022-09-28-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  op_checktemplateverify] à [SIGHASH_ANYPREVOUT][topic sighash_anyprevout]).
+  Towns prévoit également d'inclure les changements majeurs proposés à la politique
+  de relai des transactions (tel que [package relay][topic package relay]).
+
+    Towns cherche à obtenir des critiques constructives sur l'idée de cette nouvelle
+    mise en œuvre de tests, appelée [Bitcoin Inquisition][], ainsi qu'une
+    relecture des [pull requests][bi prs] ajoutées aux paramètres initiaux du
+    soft forks.
+
+## Selection de Q&R du Bitcoin Stack Exchange
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2022-09-28-newsletter.md:67]: This section is duplicated with the news section twice in this newsletter.

**Review by bitschmidty — CHANGES_REQUESTED** [review]

File: _posts/fr/newsletters/2022-09-28-newsletter.md (hunk 2 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-28-newsletter.md:67]: Done

**Review by Copinmalin — COMMENTED** [review]
section in english deleted

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Review by bitschmidty — APPROVED** [review]
Cleaned up the commit history and squashed

**Pull request merged**